simplq/src/components/pages/Home/Nav/Burger.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React, { useState } from 'react';
 import styles from '../../../../styles/homePage.module.scss';
```

Comment:
I'll remove these after the navbar is done. Eslint throws some errors otherwise.

Reply:
Yes, we can remove at the end.